eslint.config.mjs

```diff
@@ -89,6 +89,61 @@ const noNavigateUseClerk = {
   },
 };
 
+const noUnstableMethods = {
```

Comment:
New ESLint rule that should hopefully avoid any accidental addition of `__unstable_` prefixes